cmd/logs.go

```diff
@@ -71,7 +90,9 @@ func newWebserverLogsCmd() *cobra.Command {
 astro deployment logs webserver YOU_DEPLOYMENT_ID -s string-to-find
 `,
 		Args: cobra.ExactArgs(1),
-		RunE: webserverRemoteLogs,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return fetchRemoteLogs("webserver", args)
```

Comment:
I think we should use constants for Airflow components to fetch the logs for like `const LOG_WEBSERVER = "webserver"`

Reply:
moved to a constant